src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1560,6 +1560,10 @@ internal enum ErrorCode
         #region diagnostics introduced for recursive patterns
         // PROTOTYPE(patterns2): renumber these before committing
         ERR_MissingPattern = 8500,
+        ERR_InferredRecursivePatternType = 8501,
+        ERR_WrongNumberOfSubpatterns = 8502,
+        ERR_PropertyPatternNameMissing = 8503,
+        //ERR_FeatureIsUnimplemented = 8504,
```

Comment:
>//ERR_FeatureIsUnimplemented = 8504, [](start = 8, length = 36)

Please remove commented out code

Reply:
It is intentionally preserved as it corresponds to a resource key that is useful from time to time during feature development.

---
In reply to: [156509714](https://github.com/dotnet/roslyn/pull/23209#discussion_r156509714) [](ancestors = 156509714)